libs/dyn/convert/from_typed.go

```diff
@@ -142,18 +158,17 @@ func fromTypedString(src reflect.Value, ref dyn.Value) (dyn.Value, error) {
 
 		return dyn.V(value), nil
 	case dyn.KindNil:
-		// This field is not set in the reference, so we only include it if it has a non-zero value.
-		// Otherwise, we would always include all zero valued fields.
-		if src.IsZero() {
```

Comment:
nit: inverting to `src.IsZero() && !slices.Contains(options, includeZeroValues)` makes it more readable

Reply:
done